wolox-training/rof-angular
# Implemented rent view and functionalities

## Summary

Implemented rent view and functionalities

## Test Plan

* Log into your account
* Go to dashboard
* Click on the rent button on any of the books
* You will be redirected to a view with a date input and a rent button
* Rent a book
* You should be redirected to the dashboard once again.

## Trello Card 

 https://trello.com/c/egRy8jZE

## Screenshot

![rentabook](https://cloud.githubusercontent.com/assets/2911971/10830257/743ffd54-7e5d-11e5-83bf-5948367d8a1f.png)
